README.md

```diff
@@ -0,0 +1,31 @@
+# vim-dokuwiki - a VIM syntax file for DokuWiki
+
+## Install
+
+### Vim-Plug
+```
+Plug 'xerus2000/vim-dokuwiki'
```

Comment:
Please change this to `nblock/vim-dokuwiki`.

Reply:
You're right, missed that :sweat_smile: